Adwaita/resource/webkit.css

```diff
@@ -0,0 +1,242 @@
+/* 
+This file provides overrides for web based elements in the steam client
+
+You can use steam's built in inspector to find classes by launching it 
+with the `-dev` argument and hitting F12
+
+Classes will be likely randomly suffixed, you will need the `*=` 
+selector for those
+*/
+
+/* -------------- */
+/* --- Colors --- */
+/* -------------- */
+
+:root {
+    --bg: #242424;
+    --fg: #ffffff;
+    --radius: 15px;
```

Comment:
[libadwaita stylesheet](https://gitlab.gnome.org/GNOME/libadwaita/-/tree/main/src/stylesheet) uses `6px` radius for small widgets like buttons and `12px` for larger widgets like cards, rows, windows.

From https://gitlab.gnome.org/GNOME/libadwaita/-/blob/main/src/stylesheet/_common.scss:
```scss
$button_radius: 6px;
$card_radius: $button_radius + 6;
$menu_radius: 6px;
$window_radius: $button_radius + 6;
$popover_radius: $window_radius;
```

Reply:
Should be good now https://github.com/tkashkin/Adwaita-for-Steam/pull/6/commits/a698e9ff151fc4c95ad5f4238b1a146795823ed4